packages/core/src/lib/icon/__tests__/icon.spec.ts

```diff
@@ -9,6 +9,11 @@ describe('Icon', () => {
     expect(container.querySelector('svg')).toBeVisible();
   });
 
+  it('Renders an icon with the correct name attribute', () => {
+    render(Icon, { name: 'camera-outline' });
+    expect(screen.getByTestId('icon-camera-outline')).toBeInTheDocument();
```

Comment:
[Readability nit]

```suggestion
    const result = screen.getByTestId('icon-camera-outline')
    expect(result).toBeInTheDocument();
```

Reply:
23d95db 

(I will update the other ones if I see them and they look unreadable)